ehitchhiking-ui/src/app/pages/main-screen/routes-list/routes-list.component.ts

```diff
@@ -8,12 +8,31 @@ import {UtilsService} from "../../../shared/services/utils.service";
 })
 export class RoutesListComponent implements OnInit {
 	@Input() activeRoutesCollection: Partial<Route>[];
+	@Output() routeToDisplay = new EventEmitter<any>(); // TODO
+	isChecked: boolean;
 
 	constructor() {}
 
-	ngOnInit() {}
+	ngOnInit() {
+	  this.isChecked = false;
+  }
 
 	parseDate(date: Date): string {
 		return UtilsService.formatDate(date);
 	}
+
+  displayRoute(index: number) {
+    this.isChecked = !this.isChecked;
+    if (this.isChecked) {
+      this.routeToDisplay.emit(index);
+    }
+    else {
+      this.routeToDisplay.emit(-99);
```

Comment:
Magic number -99. Pls create a shared constant and change value to -1

Reply:
done